CharlieBackend.Api/Controllers/MentorsController.cs

```diff
@@ -48,6 +49,41 @@ public async Task<ActionResult> PostMentor(long accountId)
             return createdMentorModel.ToActionResult(); ;
         }
 
+        /// <summary>
+        /// Get only active students
+        /// </summary>
+        /// <response code="200">Successful return of students list</response>
+        [Authorize(Roles = "Admin, Mentor, Secretary")]
+        [HttpGet("active")]
+        public async Task<ActionResult<IList<MentorDto>>> GetAllActiveStudents()
+        {
+
+            var mentors = await _mentorService.GetAllActiveMentorsAsync();
+
+            return mentors.ToActionResult();
+        }
+
+        /// <summary>
+        /// Get mentor information by mentor id
+        /// </summary>
+        /// <response code="200">Successful return of mentor</response>
+        /// <response code="409">Error, can not find mentor</response>
```

Comment:
use 404 code when resource was not found

Reply:
done